web/wp-content/themes/dctx-v2/template-parts/content-blocks/block-wds-carousel.php

```diff
@@ -0,0 +1,77 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-ijgmkh.php, but found phpcs-scan-IjgmkH.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).